content/en/blog/2024/otel-cicd-sig/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: OpenTelemetry Is Expanding Into CI/CD Observability
+linkTitle: OpenTelemetry Is Expanding Into CI/CD Observability
+date: 2024-12-01
+author: >- 
+  [Dotan Horovits](https://github.com/horovits/) (CNCF Ambassador),
+  [Adriel Perkins](https://github.com/adrielp) (Liatrio)
+canonical_url: https://www.cncf.io/blog/2024/11/04/opentelemetry-is-expanding-into-ci-cd-observability/ 
+issue: 5546
+sig: CI/CD Observability 
+cSpell:ignore: horovits liatrio 
+---
+
+## OpenTelemetry Is Expanding Into CI/CD Observability
+
+
+We’ve been talking about the need for a common “language” for reporting and observing CI/CD pipelines for years, and finally, we see the first “words” of this language entering the “dictionary” of observability—the OpenTelemetry open specification. With the recent release of OpenTelemetry’s Semantic Conventions, v1.27.0, you can find [designated attributes for reporting CI/CD pipelines](https://opentelemetry.io/docs/specs/semconv/attributes-registry/cicd/). 
+
+This is the result of the hard work of the [CI/CD Observability Special Interest Group (SIG) within OpenTelemetry](https://github.com/open-telemetry/community/blob/main/projects/ci-cd.md). As we accomplish the core milestone for the first phase, we thought it’d be a good time to share it with the world.
+
+## Engineers need observability into their CI/CD pipelines 
+
+[CI/CD observability](https://medium.com/@horovits/fcc6c10c4987) is essential for ensuring that software is released to production efficiently and reliably. Well-functioning CI/CD pipelines directly impact business outcomes by shortening [Lead Time for Changes DORA metrics](https://horovits.medium.com/improving-devops-performance-with-dora-metrics-918b9604f8e2) and enabling fast identification and resolution of broken or flaky processes. By integrating observability into CI/CD workflows, teams can monitor the health and performance of their pipelines in real time, gaining insights into bottlenecks and areas that require improvement. 
```

Comment:
```suggestion
[CI/CD observability](https://medium.com/@horovits/fcc6c10c4987) is essential for ensuring that software is released to production efficiently and reliably. Well-functioning CI/CD pipelines directly impact business outcomes by shortening [Lead Time for Changes DORA metric](https://horovits.medium.com/improving-devops-performance-with-dora-metrics-918b9604f8e2) and enabling fast identification and resolution of broken or flaky processes. By integrating observability into CI/CD workflows, teams can monitor the health and performance of their pipelines in real time, gaining insights into bottlenecks and areas that require improvement. 
```

This was outdated, but not resolved, see https://github.com/open-telemetry/opentelemetry.io/pull/5718#discussion_r1868532420